### PR TITLE
Update PersonalizerExample and HttpRequestFeatures Samples

### PR DESCRIPTION
**Pull request by mandyshieh**
## Purpose
Update Personalizer samples.

* ...

## Does this introduce a breaking change?

```
[ ] Yes
[x ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?
Update PersonalizerExample and HttpFeaturizer to latest Personalizer nuget package, added README.

```
[ ] Bugfix
[ ] Feature
[x ] Code style update (formatting, local variables)
[x ] Refactoring (no functional changes, no api changes)
[x ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code
Run samples.
```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

File: quickstarts/csharp/PersonalizerExample/Program.cs [hunk 1]
```diff
@@ -105,7 +105,8 @@ static void Main(string[] args)
         static PersonalizerClient InitializePersonalizerClient(string url)
         {
             PersonalizerClient client = new PersonalizerClient(
-                new ApiKeyServiceClientCredentials(ApiKey)) {Endpoint = url};
+                new ApiKeyServiceClientCredentials(ApiKey))
+            { Endpoint = url };
```
Comment (tyclintw) [inline quickstarts/csharp/PersonalizerExample/Program.cs]: >            [](start = 1, length = 11)

[nit] spacing

File: quickstarts/csharp/PersonalizerExample/README.md [hunk 2]
```diff
@@ -1 +1,8 @@
 # About this Quickstart
+This sample interactively takes the time of day and the users's taste preference as context, and sends it to the Azure Personalizer instance, which then returns the top personalized food choice, along with the recommendation probability distribution of each food item.
+The user then inputs whether or not Personalizer predicted correctly, which is data used to improve Personalizer's prediction model.
```
Comment (tyclintw) [inline quickstarts/csharp/PersonalizerExample/README.md]: Did this text come from somewhere? It's a little light but a good first start. #Closed

File: samples/HttpRequestFeatures/README.md [hunk 3]
```diff
@@ -1 +1,7 @@
 # About this sample
+This sample extracts HTTP request features and adds them as context feature to a Rank requests that are sent to an Azure Personalizer instance.
+
+# Run sample
+1. Create a Personalizer instance in the Azure portal.
+2. Fill in the `ApiKey` and `ServiceEndpoint` strings in appsettings.json. These can be found in your Cognitive Services Quick start tab in the Azure portal.
```
Comment (tyclintw) [inline samples/HttpRequestFeatures/README.md]: >Fill in the `ApiKey` and `ServiceEndpoint` strings in appsettings.json [](start = 3, length = 70)

Setting API Keys in the appsettings.json is unsecure and a possible security problem during commits. I suggest sticking with the environment variable pattern in PersonalizerExample or better yet follow the pattern for `dotnet user-secrets` used in demos\PersonalizerDemo\README.md

**Review by tyclintw — CHANGES_REQUESTED** [review]
:clock1:

File: samples/HttpRequestFeatures/README.md (hunk 3 above)
Comment (mandyshieh) [reply samples/HttpRequestFeatures/README.md]: Updated both README and code

File: quickstarts/csharp/PersonalizerExample/README.md (hunk 2 above)
Comment (mandyshieh) [reply quickstarts/csharp/PersonalizerExample/README.md]: Didn't come from anywhere I just wrote it from running the sample. Added more content and detailed instructions for this one and HttpRequestFeaturizer.

**Review by tyclintw — APPROVED** [review]
:shipit:

**Pull request merged**